src/pages/TrueSightV2/pages/TokenAnalysisList.tsx

```diff
@@ -589,15 +590,20 @@ const TokenRow = ({
             <Text style={{ textTransform: 'uppercase' }}>{token.symbol}</Text>{' '}
             <RowFit gap="6px" color={theme.text}>
               {token.tokens.map(item => {
-                if (item.chain === 'ethereum') return <Icon key="eth-mono" id="eth-mono" size={12} title="Ethereum" />
-                if (item.chain === 'bsc') return <Icon key="bnb-mono" id="bnb-mono" size={12} title="Binance" />
-                if (item.chain === 'avalanche') return <Icon key="ava-mono" id="ava-mono" size={12} title="Avalanche" />
-                if (item.chain === 'polygon') return <Icon key="matic-mono" id="matic-mono" size={12} title="Polygon" />
+                if (item.chain === 'ethereum')
+                  return <Icon key={'eth-mono' + index} id="eth-mono" size={12} title="Ethereum" />
+                if (item.chain === 'bsc')
+                  return <Icon key={'bnb-mono' + index} id="bnb-mono" size={12} title="Binance" />
+                if (item.chain === 'avalanche')
+                  return <Icon key={'ava-mono' + index} id="ava-mono" size={12} title="Avalanche" />
+                if (item.chain === 'polygon')
+                  return <Icon key={'matic-mono' + index} id="matic-mono" size={12} title="Polygon" />
                 if (item.chain === 'arbitrum')
-                  return <Icon key="arbitrum-mono" id="arbitrum-mono" size={12} title="Arbitrum" />
-                if (item.chain === 'fantom') return <Icon key="fantom-mono" id="fantom-mono" size={12} title="Fantom" />
+                  return <Icon key={'arbitrum-mono' + index} id="arbitrum-mono" size={12} title="Arbitrum" />
+                if (item.chain === 'fantom')
+                  return <Icon key={'fantom-mono' + index} id="fantom-mono" size={12} title="Fantom" />
                 if (item.chain === 'optimism')
-                  return <Icon key="optimism-mono" id="optimism-mono" size={12} title="Optimism" />
+                  return <Icon key={'optimism-mono' + index} id="optimism-mono" size={12} title="Optimism" />

```

Comment:
how about this
```
return <Icon key={item.chain} id={`${item.chain}-mono`}  size={12} titlte={captialize(item.chain) }/> 
```

Reply:
will improve this batch of code later, but the key needs index to be unique between token row